### PR TITLE
Add new Article

### PR DESCRIPTION
**Pull request by harshnarang8**

### CODE REVIEW

File: _posts/2016-05-27-floss.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+[GNU Manifesto]: "https://en.wikipedia.org/wiki/GNU_Manifesto"
+
+The Linux kernel was released as a freely modifiable open source software in 1991, which was relicensed under the GNU GPL in 1992.
+FreeBSD and NetBSD were released as free software in 1993. In 1995, the Apache Web Server (A patchy web server, there is sure history behind its name :P) was released.
+
+Then came a revolutionary moment in 1998. The Netscape Corporation released its famous Netscape Communicator Suite as free software. (I am sure you guys must have heard Firefox and ThunderBird. Yeah, that is the descendant of the Netscape; however, another story, another time).
+
+Netscape saw business potential in collaborating and sharing source code, however the FSF and its ideals were not not appealing enough to them. Hence, they decided to rebrand the movement; coining the famous "open source" and the [Open Source Initiative][] was established in February, 1998. In 2001, the European Commission introduced the term FLOSS, which has been in both popular and official use since 2004.
```
Comment (sakshamsharma) [inline _posts/2016-05-27-floss.md]: Why is `Open Source Initiative` link empty? The line below is redundant, remove it, and inside the para use this:

```
[Open Source](https://opensource.org)
```

In accordance with markdown syntax.

File: _posts/2016-05-27-floss.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- gnu/linux
+image:
+	url:
+credits:
+- https://en.wikipedia.org/wiki/Free_and_open-source_software
+- https://opensource.org/licenses
+-
+---
+
+#Introduction#
```
Comment (sakshamsharma) [inline _posts/2016-05-27-floss.md]: Headings don't need the extra hash, and need a nice space for comfort

```
# Introduction
```

File: _posts/2016-05-27-floss.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+
```
Comment (sakshamsharma) [inline _posts/2016-05-27-floss.md]: Please remove this extra newline at the top

File: _posts/2016-05-27-floss.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Check out these licenses and their full texts at [The OSI website][].
+[The OSI website]: "https://opensource.org/licenses"
+
+#GNU/Linux#
+
+One can't write an article on FLOSS, and skip out GNU/Linux. There is quite a lot of history surrounding the most popular OSS Linux kernel, and it sure is a favourite amongst developers and the community alike. There are a lot of Operating systems (referred to as linux distributions) based on the linux kernel, popular ones being :
+
+##Ubuntu##
+The largest used linux OS in PCs around the world, it is quite popular amongst the people entering the dynamic world of open source. A clean graphical user interface, with the nix terminal; provide an easy transition from Windows to Ubuntu.
+Check out the [Ubuntu][] website for more details.
```
Comment (sakshamsharma) [inline _posts/2016-05-27-floss.md]: Same as above for this link

File: _posts/2016-05-27-floss.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+##Kali Linux##
+Kali Linux is a security testing distro. (Of course, it does have a cool interface :P).
+
+##Elementary OS##
+Elementary OS is a distribution based on Ubuntu, and borrows its looks from the Mac OS X.
+
+##Gentoo Linux##
+Gentoo is a distribution which, in my opinion, stands apar from others in the list. It is an OS, which you actually have to compile and install. Honestly speaking, I never had the chance to compile and install it myself, but I have seen people do it, and mind you; I have not come across an operating system so configurable during installation. For the record, I've heard it is fun using and installing. Do check out the [Gentoo][] website.
+[Gentoo]: "https://www.gentoo.org"
```
Comment (sakshamsharma) [inline _posts/2016-05-27-floss.md]: And these Gentoo links as well

**Comment by sakshamsharma** [comment]
Hi! I've added some comments to help you understand the syntax and practices even better! Should help you remember about these issues later. Sorry if you got spammed by emails.

File: _posts/2016-05-27-floss.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+9. Eclipse Public License
+
+Check out these licenses and their full texts at [the OSI website]("https://opensource.org/licenses”).
+
+## GNU/Linux
+
+One can't write an article on FLOSS, and skip out GNU/Linux. There is quite a lot of history surrounding the most popular OSS Linux kernel, and it sure is a favourite amongst developers and the community alike. 
+There are a lot of Operating systems (referred to as linux distributions) based on the linux kernel, popular ones being 
+##Ubuntu##
```
Comment (sakshamsharma) [inline _posts/2016-05-27-floss.md]: Last thing, you forgot this I guess :stuck_out_tongue: And please add a newline before Ubuntu as well.

Comment (harshnarang8) [inline _posts/2016-05-27-floss.md]: Sorry missed your note. Will do this now.

**Comment by sakshamsharma** [comment]
Cool! Will merge this in, next week, and share on pclub

**Comment by sakshamsharma** [comment]
Hi! Would merge soon, please just clear up the tags. Let them just be `open source`, `linux`; avoids cluttering the already filled up tags section. I'll post it immediately then. (Oh and also, change category to `open source`)

**Comment by harshnarang8** [comment]
Can the tags be open source, gnu/linux ?

**Comment by sakshamsharma** [comment]
Not gnu/linux, since it'd be nice to not have extra tags. There's gonna be
a linux tag, no point adding an extra one which is just gonna have one
article.

**Comment by sakshamsharma** [comment]
So I merged this by mistake :stuck_out_tongue: Let it be now. From next time, try keep Pull request and commit names/messages informative. "Add new article" isn't helpful.

**Pull request merged**